### DISCUSSION TITLE
is there anyway to hide de default device list from users

### DISCUSSION THREAD

**Discussion by IzunaMoon** [rustdesk/rustdesk-server-pro#175] [General]
i notice when you make groups dat you can add default group in de device access list

so i try making a new user to a added a few devices to dat user but i notice de default devices list so devices dat don't have a user on rustdesk pro in hour case 

but i notice de default device list still show up even if you doint give de user access to de default device list group

is this is a bug or is it meant to be like dat 
↓↓↓↓not servers↓↓↓↓
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/132285275/936e1b40-fe62-4f8e-9f1c-e3bb495ac54e)
↓↓↓↓servers ↓↓↓↓↓
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/132285275/c96efb03-b483-442c-adf3-f664948e6c9f)

servers group settings 
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/132285275/a14f22a9-4844-4fbd-8a23-268cb3a21550)
doesnt have accessto default devices so far i know

**Comment by 21pages** [comment]
> devices dat don't have a user on rustdesk pro in hour case

Unbinded (unassigned) devices do not belong to any group. After logging in, the device belongs to the logged-in user's group. All enabled users can access unbinded (unassigned) devices.
